OracleDatabase/dockerfiles/12.2.0.1/setupLinuxEnv.sh

```diff
@@ -18,6 +18,7 @@ chmod ug+x $ORACLE_BASE/$PWD_FILE && \
 chmod ug+x $ORACLE_BASE/$RUN_FILE && \
 chmod ug+x $ORACLE_BASE/$START_FILE && \
 chmod ug+x $ORACLE_BASE/$CREATE_DB_FILE && \
+chmod ug+x $ORACLE_BASE/$CHECK_DB_FILE && \
```

Comment:
Couldn't all these chmods be updated to a single call `chmod ug+x $ORACLE_BASE/*.sh`